src/kibana/components/vislib/lib/chart_title.js

```diff
@@ -32,7 +32,11 @@ define(function (require) {
      * @returns {D3.Selection|D3.Transition.Transition} DOM element with chart titles
      */
     ChartTitle.prototype.render = function () {
-      return d3.select(this.el).selectAll('.chart-title').call(this.draw());
+      var $el = d3.select(this.el).select('.chart-title').node();
```

Comment:
Using the `$` before an element typically denotes a jQuery-wrapped element, but here it doesn't seem like that's what you have. Do you mind just changing it to `el`? :)